### ISSUE TITLE
Suggestion-Home Page with basic template

### ISSUE THREAD

**Issue by NileshPatel17** [amite/js-links#3]
Can we have home page with header and footer including some basic 2-3 navigation links in boilerplate? Say-Home, Contact and About page.

**Comment by akshaynaik404** [comment]
Home, contact and about pages are not always  necessary for a Web apps home page - our '/' route after logging in, they are included mostly in a products landing page. 

Reasons : because we really don't want some one to contact us, that is not our primary goal. 

Though we can have an about page,  but it shouldn't be in the main UI. 
What the application does should be explained at the time of user onboarding (Signing Up). 

Users will visit this app,  either to post new links or engage with existing links in the UI. So 'add new links + btn' will be our main cta, along with user submitted links which will be shown in a list - as shown in the includes screenshots.

**Comment by amite** [comment]
Agreed but it would still be a useful exercise to do in a separate branch to demonstrate how react router works with react components. Beginners to react will appreciate how easy it is to make new page-like content

**Comment by akshaynaik404** [comment]
Yes,  that would be great. We can even have landing page that explains what the app is and with links to the JavaScripters community, this repo, contributors, etc...

**Comment by NileshPatel17** [comment]
My suggestion was specific to boilerplate. Going forward we can have home and about page with some basic stylesheet to kick start new project. By having this, we will have routing as part of boilerplate.

**Comment by akshaynaik404** [comment]
I understand what you mean @NileshPatel17 and I agree with you, it will be better to understand for newcomers if routing is visible by default in the boilerplate.
